Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Templates/LambdaFunctionTemplate.tt

```diff
@@ -60,7 +60,7 @@ this.Write(new FieldsAndConstructor(_model).TransformText());
         this.Write(apiParameters.TransformText());
         this.Write(new APIGatewayInvoke(_model, apiParameters.ParameterSignature).TransformText());
     }
-    else if (_model.LambdaMethod.Events.Count == 0)
+    else
```

Comment:
Currently we only support 2 event types - `APIGatewayEvents` and `SQSEvents`. 

Since `SQSEvents` does not require any special code generation, the generated method body will be same as when the Lambda method contains no events.